Projects/Feature/SearchFeature/Sources/ViewController/SearchViewController.swift

```diff
@@ -35,20 +44,20 @@ public final class SearchViewController: UIViewController {
     
     private let magniImage: UIImageView = {
         let symbolName = "magnifyingglass"
-
+        
         var configuration = UIImage.SymbolConfiguration(pointSize: 8,
                                                         weight: .light)
         configuration = configuration.applying(UIImage.SymbolConfiguration(
-                            font: UIFont.systemFont(ofSize: 20, weight: .light),
-                            scale: .default))
-
+            font: UIFont.systemFont(ofSize: 20, weight: .light),
+            scale: .default))
+        
         let migImage = UIImage(
             systemName: symbolName,
             withConfiguration: configuration)?.withTintColor(.black)
```

Comment:
가독성을 위해 형태변경 제안드립니다.
```suggestion
        configuration = configuration.applying(
            UIImage.SymbolConfiguration(
                font: UIFont.systemFont(ofSize: 20, weight: .light),
                scale: .default
            )
        )
        let migImage = UIImage(
             systemName: symbolName,
             withConfiguration: configuration
        )?.withTintColor(.black)
```